CHANGES.txt

```diff
@@ -2,6 +2,63 @@
 Pyface Changelog
 ================
 
+Release 7.3.0
+=============
+
+The most significant change introduced in this minor release is the
+replacement of Traits ``on_trait_change``-based listeners with the new Traits
+observation framework.
+
+This release is only compatible with Traits >= 6.2.0 i.e. the most recent
+version of Traits.
+
+Highlights of this release
+--------------------------
+
+* Replace use of Traits ``on_trait_change`` with ``observe``.
+
+Detailed changes
+----------------
+
+Thanks to:
+
+* Aaron Ayres
+* Kit Yan Choi
+* Rahul Poruri
+* Corran Webster
+
+Fixes
+
+* Fix undo/redo merge bugs. (#885)
```

Comment:
Note that this PR hasn't been merged yet - #885

Reply:
Merged!